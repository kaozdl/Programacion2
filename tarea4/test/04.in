# pila
apilar .
desapilar 1
apilar uno .
desapilar 1
apilar uno .
apilar dos tres cuatro cuatro .
desapilar 2
desapilar 1
apilar tres cuatro cinco seis siete ocho nueve diez .
apilar 11 12 13 14 15 16 17 18 19 MAX_PILA 21 22 .
desapilar 1
desapilar 19
desapilar 1
Fin
