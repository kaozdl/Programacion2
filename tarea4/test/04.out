1>
2>3>4>5>uno
6>7>8>cuatro
9>tres
10>11>MAX_PILA
12>MAX_PILA
13>uno
14>15>Fin.
